.travis.yml

```diff
@@ -108,6 +112,20 @@ before_script:
                   sudo make install
               )
           fi
+          if [ "$TRAVIS_OS_NAME" = linux -a "$BORINGSSL" ]; then
+            (git clone https://boringssl.googlesource.com/boringssl &&
+            cd boringssl &&
+            mkdir build &&
+            cd build &&
+            cmake -DCMAKE_INSTALL_PREFIX:PATH=$HOME/build-boringssl .. &&
+            make &&
```

Comment:
Using *cmake* with this option is probably what most maintainers use:
```bash
cmake -DCMAKE_BUILD_TYPE=release …
```

… and then you could decrease compilation time by (replace *ninja* by *make*; don't forget *libdecrepit.a*):

```bash
ninja crypto/libcrypto.a ssl/libssl.a decrepit/libdecrepit.a
```

FYI, you could build everything as shared libraries, but I don't think this makes a difference for the tests.

Reply:
> libdecrepit.a

What's that and what do we need that for?

Reply:
»Old« algorithms are collected in that library. For example, DSA and RC4.

Reply:
And regarding ninja: I dread to add more dependencies (problems) to the annoying travis environment. Besides, make vs ninja can't make *that* big difference when building from scratch in a single-core one-by-one build surely?

Reply:
Never used *make* here, but I think *make* accepts what-to-build as arguments like *ninja* does.

(I guess you want to use `make -j$(nproc) …` here.)

Reply:
Tried it, `make -j$(nproc)` works, using *make* you cannot compile a subset of files like when using *ninja*.

Reply:
> using make you cannot compile a subset of files like when using ninja.

so you're saying the generated makefile builds too many files?

Reply:
Yes, but there's no harm in that. Other than some cpu cycles.

Reply:
we can consider changing that in phase two, just now I want to focus on getting this to work...